src/components/bottomSheet/BottomSheet.tsx

```diff
@@ -1393,9 +1444,9 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
          */
         animatedKeyboardHeightInContainer.value = $modal
           ? Math.abs(
-              _keyboardHeight -
-                Math.abs(bottomInset - animatedContainerOffset.value.bottom)
-            )
+            _keyboardHeight -
+            Math.abs(bottomInset - animatedContainerOffset.value.bottom)
+          )
```

Comment:
Wasn't this also correctly indented before?

Reply:
Same as the previous comment, locally it's indented.